Makefile

```diff
@@ -116,7 +116,8 @@ config: certs ## Set up default config
 
 .PHONY: musl
 musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+	rustup target list | grep -qc $(uname_m)-unknown-linux-musl || \
```

Comment:
This line is really close. But:

1. grep takes a string for the pattern input (run `grep --help`)
2. when you run `rustup target list` you'll notice it lists all targets (not just those that are installed). The ones that are installed are specifically indicated